### PR TITLE
tests: fix error trying to create the extra-snaps dir which already exists

### PR DESCRIPTION
**Pull request by sergiocazzolato**
The test tests/nested/manual/core20-cloud-init-maas-signed-seed-data is
failing trying to create the extra-snaps directory which is being
created while the test is prepared.

+ mkdir
/home/gopath/src/github.com/snapcore/snapd/tests/nested/manual/core20-cloud-init-maas-signed-seed-data/extra-snaps
mkdir: cannot create directory
‘/home/gopath/src/github.com/snapcore/snapd/tests/nested/manual/core20-cloud-init-maas-signed-seed-data/extra-snaps’:
File exists

### CODE REVIEW

File: tests/nested/manual/core20-cloud-init-maas-signed-seed-data/task.yaml [hunk 1]
```diff
@@ -58,10 +58,6 @@ prepare: |
   cp "$TESTSLIB"/assertions/developer1.account "$NESTED_FAKESTORE_BLOB_DIR/asserts"
   cp "$TESTSLIB"/assertions/developer1.account-key "$NESTED_FAKESTORE_BLOB_DIR/asserts"
 
-  # modify and repack gadget snap to add a defaults section and use our own
-  # prepare-device hook to use the fakedevicesvc
```
Comment (stolowski) [inline tests/nested/manual/core20-cloud-init-maas-signed-seed-data/task.yaml:62]: I think comment should stay?

**Review by stolowski — APPROVED** [review]
Looks good, but I think the comment should be preserved.

File: tests/nested/manual/core20-cloud-init-maas-signed-seed-data/task.yaml (hunk 1 above)
Comment (anonymouse64) [reply tests/nested/manual/core20-cloud-init-maas-signed-seed-data/task.yaml:62]: yes please keep the comment

File: tests/nested/manual/core20-cloud-init-maas-signed-seed-data/task.yaml [hunk 2]
```diff
@@ -58,10 +58,6 @@ prepare: |
...
-  mkdir "$(nested_get_extra_snaps_path)"
```
Comment (bboozzoo) [inline tests/nested/manual/core20-cloud-init-maas-signed-seed-data/task.yaml:63]: hm couldn't this be mkdir -p then? (while keeping the comment too)

File: tests/nested/manual/core20-cloud-init-maas-signed-seed-data/task.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply tests/nested/manual/core20-cloud-init-maas-signed-seed-data/task.yaml:62]: Comment added back, thanks for the review

**Pull request merged**